### PR TITLE
Update comment of template for doc

### PR DESCRIPTION
**Pull request by bianpengyuan**
Update comments of all compiled in templates to use the new instance schema. This is for istio.io doc update.

### CODE REVIEW

File: mixer/template/logentry/template.proto [hunk 1]
```diff
@@ -26,25 +26,27 @@ syntax = "proto3";
...
-//     sourceUser: source.principal | ""
-//     method: request.method | ""
-//     url: request.path | ""
-//     protocol: request.scheme | "http"
-//     responseCode: response.code | 0
-//     responseSize: response.size | 0
-//     requestSize: request.size | 0
-//     latency: response.duration | "0ms"
-//   monitored_resource_type: '"UNSPECIFIED"'
+//   compiledTemplate: accesslog
```
Comment (kyessenov) [inline mixer/template/logentry/template.proto]: logentry

File: mixer/template/logentry/template_handler_service.proto [hunk 2]
```diff
@@ -23,25 +23,27 @@ syntax = "proto3";
...
-//     sourceUser: source.principal | ""
-//     method: request.method | ""
-//     url: request.path | ""
-//     protocol: request.scheme | "http"
-//     responseCode: response.code | 0
-//     responseSize: response.size | 0
-//     requestSize: request.size | 0
-//     latency: response.duration | "0ms"
-//   monitored_resource_type: '"UNSPECIFIED"'
+//   compiledTemplate: accesslog
```
Comment (kyessenov) [inline mixer/template/logentry/template_handler_service.proto]: logentry

Comment (bianpengyuan) [reply mixer/template/logentry/template_handler_service.proto]: Done.

File: mixer/template/logentry/template.proto (hunk 1 above)
Comment (bianpengyuan) [reply mixer/template/logentry/template.proto]: Done.

**Comment by geeknoid** [comment]
Note that for 1.2, the doc site is pulling all this stuff from the istio/istio release-1.2 branch. So if you'd like these updates in the 1.2 doc set, you'll need to migrate these changes to the istio/istio release-1.2 branch.

**Comment by bianpengyuan** [comment]
@geeknoid Yeah, thanks for the reminder!

**Pull request merged**